tests/components/august/test_sensor.py

```diff
@@ -255,6 +265,60 @@ async def test_lock_operator_remote(hass: HomeAssistant) -> None:
     )
 
 
+async def test_lock_operator_manual(hass: HomeAssistant) -> None:
+    """Test operation of a lock with doorsense and bridge."""
+    lock_one = await _mock_doorsense_enabled_august_lock_detail(hass)
+
+    activities = await _mock_activities_from_fixture(
+        hass, "get_activity.lock_from_manual.json"
+    )
+    await _create_august_with_devices(hass, [lock_one], activities=activities)
+
+    entity_registry = er.async_get(hass)
+    lock_operator_sensor = entity_registry.async_get(
+        "sensor.online_with_doorsense_name_operator"
+    )
+    assert lock_operator_sensor
+    assert (
```

Comment:
Please fetch the state once instead of calling `hass.states.get()` multiple times

Reply:
These are used the same way in all the old tests as well.  Should I change them there at the same time, or should I leave that to another PR?

Reply:
It would be great to clean up the older tests in followup PR

Reply:
I'll do that once this is merged